notebooks/tel_and_site/subsys_req_ver/m1m3/SITCOM-1136_M1M3_analysis_stability_tracking.ipynb

```diff
@@ -0,0 +1,704 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #63.        rms = filtered_correctedVariablePlot.rolling(rolling).std()
</pre><p> </p><p>Have you tested wether this rolling std impact or not the final statistics of the total number of unstable events? Or if the window you selected will impact it?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_vandv/pull/107/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>I should try different rolling. I have only done it for 10 because it was the one we used when we analysed the stability in the slews.&nbsp;</p><p> </p><p>I can do some tests and let you know.</p>

Reply:
<p>I've tested the results when the rolling is 5 and 20. The number of events that are unstable is very similar in all the cases. And (the results more important), the inestability happens in the first seconds of the trying in all the cases. </p><p> </p><p><strong>Position  |  roll = 10  | roll=20   |  roll=5  |</strong></p><p>x Pos         |       33         |    32         |        32    |</p><p>y Pos         |         0         |       3         |         0     |</p><p>z Pos         |         0         |       0         |         0     |</p><p>x Rot         |       19         |    18         |        15    |</p><p>y Rot         |       22         |    21         |        20    |</p><p>z Rot         |       26         |    25         |        26    |</p>

Reply:
<p>Thanks for checking this!</p>